workflow/scripts/export_ariadne_variables.py

```diff
@@ -1948,6 +2017,21 @@ def get_prices(n, region):
 
     var = pd.Series()
 
+    kwargs = {
+        'groupby': n.statistics.groupers.get_name_bus_and_carrier,
+        'nice_names': False,
+    }
+
+    # co2 additions
+    co2_price = -n.global_constraints.loc["CO2Limit", "mu"] - n.global_constraints.loc["co2_limit-DE", "mu"]
+    # specific emissions in tons CO2/MWh (https://www.volker-quaschning.de/datserv/CO2-spez/index_e.php)
+    specific_emisisons = {
+        "oil" : 266.5e-3,
+        "gas" : 200.8e-3,
+        "hard coal" : 338.2e-3,
+        "lignite" : 398.7e-3,
+    }
+
```

Comment:
probably the values from this source are similar, but we should try to use the factors defined internally in the model (efficiency) whenever possible